fms/utils/serialization.py

```diff
@@ -399,13 +401,17 @@ def load_state_dict_into_model(
     """
 
     # 1. Get the adapter from checkpoint sd to fms sd
+    # TODO: could create custom source based on the unfuse_strategy, and call corresponding adapter
+    # TODO: adapter now always convert to fused... need a separate one or customize this
+    # source="gptq_hf_unfused"  # !!! DEBUG: hardcode source
```

Comment:
clean up these comments, see my suggestion just up on how to handle the fused/unfused ckpts